### ISSUE TITLE
Cool utility - still worth updating to make work again?

### ISSUE THREAD

**Issue by tattwamasi** [1200wd/bulkpaperwallets#3]
Hi, I've been trying to get the project running, and found that it was written against an old version of Bitcoinlib.  I started doing naive updates to get it going, and got things partially working, but keep hitting one more error as I go through different code paths.  I am afraid I'm not doing things properly since I'm not really up to speed on the code. Before spending a lot more time on it, I'm just curious to get a feel of the overall status of it.

Should this be as simple as adapting to a few changed interfaces in bitcoinlib, or are there fundamental library changes or just problems with the approach vs. better alternatives nowadays?  I looked around and couldn't find another batch util like this that had the same capabilities, though can do it by combining the paper wallet bulk step with a separate batch funding step.  Still seems like a cool util

**Comment by mccwdev** [comment]
Yes, this code is a bit outdated, since there where a lot of changes in the bitcoin library the last 2 years. But the fundamentals of the library are the same, so wallet creation, fee estimation, updating work more-or-less the same but should be a bit more efficient.

Maybe you can create a PR with your changes and fixes and let me know if there are some issues still. Then I can run some tests and see if it need some more fixes. Would be nice update and make this repo work again.

**Comment by tattwamasi** [comment]
Sounds reasonable.  I'll keep going and see what happens.  Thanks for the reply
